src/DynamoCore/Engine/CodeGeneration/AstBuilder.cs

```diff
@@ -498,8 +444,7 @@ private void CompileToAstNodes(NodeModel node, List<AssociativeNode> resultList,
         /// <param name="nodeGuid"></param>
         private void OnAstNodeBuilding(Guid nodeGuid)
         {
-            if (nodeContainer != null)
-                nodeContainer.OnCompiling(nodeGuid);
+            nodeContainer?.OnCompiling(nodeGuid);
```

Comment:
hmm, maybe I am confused, did you mean to use `??` or is this just a syntax for null coalescing that also works?

Reply:
ahh cool:
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-conditional-operators